yoyodyne/models/expert.py

```diff
@@ -17,12 +17,17 @@
 from torch.utils import data
 
 from .. import defaults
+from ..data import indexes
```

Comment:
Do this as `from .. import data` then use `data.Index`. That's why I did [this](https://github.com/CUNY-CL/yoyodyne/blob/9cea4d4d483c3b473b6d5a5cdc0abd1a05095419/yoyodyne/data/__init__.py#L8).